doc/source/user_guide/index.rst

```diff
@@ -13,22 +13,26 @@ You can change these global settings at anytime using the following functions:
 
 .. code-block:: python
```

Comment:
pycon or python?

Reply:
@MaxJPRey So I guess we should update this one too? Shouldn't we?

Reply:
No this one is OK, because it is pure Python code, without the `>>>` in front of each line.

Reply:
I could do that because we are not displaying any output from a `print`.

Reply:
@MaxJPRey Ok thanks a lot for the clear explanation